shared/hoverAlerts.ts

```diff
@@ -0,0 +1,84 @@
+import * as sourcegraph from 'sourcegraph'
+
+export const HoverAlerts = {
+    LSIF: <sourcegraph.Badged<sourcegraph.HoverAlert>[]>[
+        {
+            summary: {
+                kind: sourcegraph.MarkupKind.Markdown,
+                value:
+                    'Semantic result. [Learn more.](https://docs.sourcegraph.com/user/code_intelligence/lsif)',
```

Comment:
In context on the hover I think these make sense, but reading them here I'm wondering if we could be more clear what this means. Not everyone will understand what a language server is, what semantic means in this context, etc. I know some of this will be clarified in the hover info, but am curious to briefly discuss.

Reply:
There's a thread here where we were trying to brainstorm how to get this across, not much luck: https://sourcegraph.slack.com/archives/CHXHX7XAS/p1589989321162100

The text before 'Learn more.' would need to be max ~4 words for the alert to fit on one line, which I would really like.

Reply:
Agree, and I remember that thread. This seems like the best we have for now!